Database-Features/removing-corrupt-data.md

```diff
@@ -0,0 +1,91 @@
+# Identifying and Removing Corrupt Data
+
+## Scope
+
+In very rare occasions, a chain of actions may cause some data in the database to become corrupt. This article gives some tools you can use to help identify if there is corrupt data or not. 
+
+**Corrupt data occurs very rarely, even if your issues match the symptoms in this article. You should only follow the below instructions after discussion with Exasol support.**
+
+## Diagnosis
+
+If something in the database is corrupt, it can manifest itself in several different ways. For example:
+* Some queries may cause the database to suddenly restart. The queries may reference certain objects in the database which are corrupt.
+* Backups are unable to be written.
+
+## Explanation
+
+Data is stored column-wise in Exasol, which means that if certain data blocks are removed or not accessible, then any attempt to access that block will crash the database. This includes queries reading certain columns, indexes, statistics, or other data structure, as well as backups. In this case, the database is in an inconsistent state and requires intervention.
```

Comment:
This beginning is a bit confusing. What about
physical data stored in blocks -> blocks belong to columns, indices or internal objects as Exasol is column-oriented database -> ...

Reply:
Updated